Tests/TracingInstrumentationTests/SpanTests.swift

```diff
@@ -175,13 +179,19 @@ final class SpanTests: XCTestCase {
             onEnd: { _ in }
         )
 
+        #if compiler(>=5.2)
         var attributes = SpanAttributes()
         attributes.sampleHttp.statusCode = 418
+        #else
+        let attributes = SpanAttributes()
```

Comment:
the corresponding not typesafe fancy version here?

```
attributes["http.status_code"] = .int(418)
```

would it be?

Reply:
Added ✅